src/main/resources/com/rultor/agents/req/merge.sh

```diff
@@ -20,6 +20,12 @@ while [ $(git show-branch "${BRANCH}" 2>/dev/null | wc -l) -gt 0 ]; do
 done
 
 git checkout -B "${BRANCH}" "fork/${fork_branch}"
+git fetch origin
+if git log --name-status --oneline HEAD..origin/${head_branch} | grep ".rultor.yml"; then
+  echo "Since original repository has different version of .rultor.yml file you have to sync your fork with the upstream repository."
+  exit 1
+fi
```

Comment:
@mkordas Yes, your fork cannot be behind, if syncing leads to update `.rultor.yml`. Otherwise we can have situation: master is OK, branch is OK, but after merge build is broken. For example: you make a fork, and add new script. After that master `.rultor.yml` is updated (new script). Both master and your branch is OK. But merged result fails (because this two script can be incompatible). If `.rultor.yml` is synced then this situation will not occurs.